test-infra/src/main/java/com/datastax/oss/driver/api/testinfra/ccm/CcmBridge.java

```diff
@@ -232,6 +234,45 @@ private static boolean isWindows() {
     return System.getProperty("os.name", "").toLowerCase(Locale.US).contains("win");
   }
 
+  private static Version parseCcmVersion() {
+    String versionString = CCM_VERSION_PROPERTY;
+    Version result = null;
+    try {
+      result = Version.parse(versionString);
+      return result;
+    } catch (IllegalArgumentException ex) {
+      LOG.warn(
+          "Failed to parse ccm.version property \'{}\' as Version instance. Attempting to fetch it through \'ccm node versionfrombuild\'",
+          versionString);
+    }
+    Path configDir = null;
+    try {
+      configDir = Files.createTempDirectory("ccmParseVer");
```

Comment:
Last question, will it fail if directory is already there ?

Reply:
@Bouncheck , ping

Reply:
Looks like it to me, but it's extremely unlikely, since the name contains random long